build.sbt

```diff
@@ -217,6 +220,45 @@ lazy val `sbt-exercise` = (project in file("sbt-exercise"))
   )
   .enablePlugins(BuildInfoPlugin)
 
+// Distribution
+
+lazy val gpgFolder = sys.env.getOrElse("SE_GPG_FOLDER", ".")
+
+lazy val publishSettings = Seq(
+  organizationName := "Scala Exercises",
+  organizationHomepage := Some(new URL("http://scala-exercises.org")),
+  startYear := Some(2016),
+  description := "Scala Exercises: The path to enlightenment",
+  homepage := Some(url("http://scala-exercises.org")),
+  pgpPassphrase := Some(sys.env.getOrElse("SE_GPG_PASSPHRASE", "").toCharArray),
+  pgpPublicRing := file(s"$gpgFolder/pubring.gpg"),
+  pgpSecretRing := file(s"$gpgFolder/secring.gpg"),
+  credentials += Credentials("Sonatype Nexus Repository Manager",  "oss.sonatype.org",  sys.env.getOrElse("PUBLISH_USERNAME", ""),  sys.env.getOrElse("PUBLISH_PASSWORD", "")),
+  scmInfo := Some(ScmInfo(url("https://github.com/scala-exercises/site"), "https://github.com/scala-exercises/site.git")),
+  licenses := Seq("Apache License, Version 2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt")),
+  publishMavenStyle := true,
+  publishArtifact in Test := false,
+  pomIncludeRepository := Function.const(false),
```

Comment:
Given that  `pomIncludeRepository` expect a function that returns a boolean, I used  `Function.const` which represents the constant function:

```scala
def const[T, U](x: T)(y: U): T = x
```

Actually we can express it thus:

```scala
pomIncludeRepository := { _ => false }
```